src/controllers/view/privateController.js

```diff
@@ -0,0 +1,82 @@
+const Blogs = require("../../models/Blogs");
+const Comments = require("../../models/Comments");
+const Users = require("../../models/Users");
+
+const renderDashboard = async (req, res) => {
+  try {
+    const { loggedIn, user } = req.session;
+    // console.log(req.session);
+    // console.log(user);
+    // console.log(loggedIn);
+    let blogData = await Blogs.findAll({
+      include: [{ model: Comments, as: "comments" }],
+      where: {
+        user_id: user.id,
+      },
+    });
+    blogData = blogData.map((each) => {
+      return each.get({ plain: true });
+    });
+
+    // console.log(blogData);
+
+    res.render("dashboard", { loggedIn, blogData });
+  } catch (err) {
+    res.status(500).json(err);
+  }
+};
+
+const renderBlogs = async (req, res) => {
+  try {
+    const { loggedIn, user } = req.session;
+    // console.log(req.session);
```

Comment:
Remember to remove console logs. There are a number of them throughout the code, some commented out and some not.